src/docs/product/dashboards/index.mdx

```diff
@@ -0,0 +1,68 @@
+---
+title: Dashboards
+sidebar_order: 6
+redirect_from:
+  - /workflow/visibility/
+  - /workflow/dashboards/
+  - /product/error-monitoring/dashboards/
+description: "Sentry's Dashboards help you gain detailed perspectives of your application's health by providing filtering tools to navigate your issues across multiple projects in a single view."
+---
+
+Dashboards help you gain detailed perspectives of your application's health by providing visibility to your errors and transactions across multiple projects in a single view.
+
+Dashboards are comprised of one or more dashboard widgets. Each widget is a visualization of one or more [Discover queries](/product/discover-queries/).
+
+## Overall View of Application
+
+![Widgets visualizing events, errors by country, affected users, and handled vs unhandled issues.](dashboard.png)
+
+When visiting Dashboards for the first time, your organization is provided a set of pre-built widgets. You may edit them to suit your organization's needs.
+
+## Customizing widgets
+
+<Alert title="Note" level="info">
+  Available if your organization is on either the Business plan or the Trial plan.
+</Alert>
+
+![Widget builder.](widget_builder.png)
+
+You can build or edit widgets by:
+
+1. Clicking on "Edit Dashboard" button on the top right hand corner.
+2. Either click on the pencil icon to edit a widget, or clicking on the widget panel with the plus icon.
+
+This will open up the widget builder (see screenshot above).
+
+A widget consists of the following components:
+
+- Widget name
+- Visualization types
+- One or more Discover queries
+  - Search conditions
+  - Fields, tags, or functions
+
+### Visualization types
+
+**Note:** Only functions are available as choices for:
+
+- Area chart
+- Bar chart
+- Line chart
+- World Map
+- Big Number
+
+#### Area, Bar, and Line Charts
+
+Displays functions in various timeseries chart types. 
+
+#### Table Results
+
+Display top 5 row results of a [Discover query](/product/discover-queries/).
+
+#### World Map
+
+Events are queried using the given function and `geo.country_code`. The results are overlaid on the world map.
+
+#### Big Number 
+
+Displays the current value of a single function.
```

Comment:
Can we clarify which function? Is it based on the selection above?
```suggestion
Displays the current value of a single function.
```

Reply:
Only one function can be chosen for a big number widget.